common/flatpak-installation.c

```diff
@@ -2660,3 +2665,32 @@ flatpak_installation_prune_local_repo (FlatpakInstallation *self,
 
   return flatpak_dir_prune (dir, cancellable, error);
 }
+
+/**
+ * flatpak_installation_run_triggers:
+ * @self: a #FlatpakInstallation
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Run the trigger commands to update the files exported by the apps in
+ * @self. Should be used after one or more app install, upgrade or
+ * uninstall operations with the #FLATPAK_INSTALL_FLAGS_NO_TRIGGERS,
+ * #FLATPAK_UPDATE_FLAGS_NO_TRIGGERS or #FLATPAK_UNINSTALL_FLAGS_NO_TRIGGERS
```

Comment:
Nitpick: IIRC, gtk-doc wants enum members to be prefixed by `%`. `#` is for type names.